scripts/run_driver_tests.psm1

```diff
@@ -46,19 +46,28 @@ function Invoke-NetshEbpfCommand
 function Invoke-Test
 {
     param([Parameter(Mandatory=$True)][string] $TestName,
-          [Parameter(Mandatory=$True)][bool] $VerboseLogs)
+          [Parameter(Mandatory=$True)][bool] $VerboseLogs,
+          [Parameter(Mandatory=$False)][bool] $Coverage)
 
     Write-Log "Executing $Testname"
 
     $LASTEXITCODE = 0
 
+    $OriginalTestName = $TestName
+    $ArgumentsList = @()
+
+    if ($Coverage) {
+        $ArgumentsList += @('--modules=C:\eBPF', '--export_type', ('binary:' + $TestName + '.cov'), '--', $TestName)
+        $TestName = 'C:\Program Files\OpenCppCoverage\OpenCppCoverage.exe'
```

Comment:
Suggestion: Can you put the path ("C:\Program Files\OpenCppCoverage\") in a local variable that can be used here, line 88, line 108, etc?